### PR TITLE
Merge main into release branch

### PR DESCRIPTION
**Pull request by Top-Cat**

### CODE REVIEW

File: src/shared/web-components/src/FamilyHubs.SharedKernel.Razor/Cookies/ICookiePage.cs [hunk 1]
```diff
@@ -6,6 +6,14 @@ public interface ICookiePage
...
     string CookiePolicyContent { get; }
     bool ShowSuccessBanner { get; }
+    ConsentCookie? Cookie { get; }
 
+    void OnGet(HttpRequest request);
     void OnPost(bool analytics, HttpRequest request, HttpResponse response);
-}
\ No newline at end of file
+
+    public record ConsentCookie
```
Comment (stevesatdfe) [inline src/shared/web-components/src/FamilyHubs.SharedKernel.Razor/Cookies/ICookiePage.cs:14]: Can this not be written as:

public record ConsentCookie(bool analytics, int version);

Result in the same thing?

File: src/ui/connect-ui/src/FamilyHubs.Referral.Core/ApiClients/OrganisationClientService.cs [hunk 2]
```diff
@@ -149,7 +149,7 @@ public async Task<List<KeyValuePair<TaxonomyDto, List<TaxonomyDto>>>> GetCategor
 
     private static string GetPositionUrl(string? serviceType, double? latitude, double? longitude, double? proximity, string status, int pageNumber, int pageSize)
     {
-        return $"api/services-simple?serviceType={serviceType}&status={status}&pageNumber={pageNumber}&pageSize={pageSize}&isFamilyHub=false{(
+        return $"api/services-simple?serviceType={serviceType}&status={status}&pageNumber={pageNumber}&pageSize={pageSize}{(
```
Comment (stevesatdfe) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Core/ApiClients/OrganisationClientService.cs:152]: Maybe a StringBuilder is a better option here as this looks really hard to follow what is going on building the URL up?

**Review by stevesatdfe — CHANGES_REQUESTED** [review]

File: src/shared/web-components/src/FamilyHubs.SharedKernel.Razor/Cookies/ICookiePage.cs (hunk 1 above)
Comment (stevesatdfe) [reply src/shared/web-components/src/FamilyHubs.SharedKernel.Razor/Cookies/ICookiePage.cs:14]: Fair enough. Forgot about where it is used.

File: src/ui/connect-ui/src/FamilyHubs.Referral.Core/ApiClients/OrganisationClientService.cs (hunk 2 above)
Comment (Top-Cat) [reply src/ui/connect-ui/src/FamilyHubs.Referral.Core/ApiClients/OrganisationClientService.cs]: We already had most of one, done

**Review by stevesatdfe — APPROVED** [review]
